packages/ott/ott.0.27/files/ott.install

```diff
@@ -0,0 +1,3 @@
+bin: ["src/ott"]
+doc: ["doc/ott_manual_0.25.pdf" "doc/ott_manual_0.25.html"]
+share: ["emacs/ott-mode.el" {"emacs/ott-mode.el"} "tex/ottlayout.sty" {"tex/ottlayout.sty"}]
```

Comment:
Wouldn't it be something that going with ```share_root``` instead of ```share``` ? (at least the emacs mode)